templates/formSection.hbs

```diff
@@ -0,0 +1,39 @@
+<section class="container__form">
+
+    <div class="container__info-form  component"> 
+
+        <div class="info-form">
+            <div class="info-form__paragraph">
+                <h1>{{titleText}}</h1>
+                <p>{{infoParagraph}}</p>
+            </div>
+
+            <div class="login-form">
+                <form action="/login" method="post">
+
+                    <div class="form-control">
+                        <input type="text" name="nombre" placeholder="Nombre "  required />
+                    </div>
+
+                    <div class="form-control">   
+                        <input type="email" name="Correo" placeholder="Correo"  required />
```

Comment:
don't use upper case in the name of an input. please replace by `correo`

Reply:
Done